app/services/actual/import.rb

```diff
@@ -22,14 +22,19 @@ def initialize(report:, uploader:)
 
     def import(actuals)
       ActiveRecord::Base.transaction do
-        @imported_actuals = actuals.map.with_index { |row, index| import_row(row, index) }
+        @imported_actuals = actuals.map.with_index { |row, index| import_row(row, index) }.compact
+
+        @errors = [] if @imported_actuals.present? && has_only_default_value_errors?
```

Comment:
I wondered if this reassignment of `@errors` could be avoided, but I'm having problems figuring out if it's possible without too much refactoring. 

I think it might be nicer if we could only assign an error in this scenario if comment is present. However, it seems like it would be a pain to assign errors if comment is not present and other errors are present

Reply:
Yeah, it's that latter thing that makes it messy, if I understand you correctly! We still want the error if there are no comments and no valid rows

I think it might be nice to do a further refactor/design to draw out the comments scenarios in the error messaging itself though